code/modules/roguetown/roguecrafting/structure.dm

```diff
@@ -40,36 +40,75 @@
 	result = /obj/structure/fluff/psycross/crafted
 	reqs = list(/obj/item/grown/log/tree/small = 1,
 				/obj/item/grown/log/tree/stake = 3)
-	verbage = "carpent"
-	verbage_tp = "carpents"
-	craftsound = 'sound/foley/Building-01.ogg'
-
+	verbage = "construct"
+	verbage_tp = "constructs"
 
 /datum/crafting_recipe/roguetown/structure/pyre
 	name = "wooden pyre"
 	result = /obj/machinery/light/rogue/campfire/pyre
 	reqs = list(/obj/item/grown/log/tree/small = 2,
 				/obj/item/grown/log/tree/stake = 3)
+	verbage = "construct"
+	verbage_tp = "constructs"
+
+/datum/crafting_recipe/roguetown/structure/swing_door
+	name = "swing door"
```

Comment:
this is gonna have to wait as structure crafting is being reworked and merged soon

Reply:
okay structure slapcrafting has been merged please refer to that for adding in new structure crafts this method only exists currently for survival structures like fires